### PR TITLE
Plane: update FAQ: Links, BadAHRS, and Autotrim

### PR DESCRIPTION
**Pull request by Hwurzburg**
Addresses issue  #2119 and more

### CODE REVIEW

File: plane/source/docs/fixed-wing-faq.rst [hunk 1]
```diff
@@ -228,26 +230,26 @@ sensor data. The techniques are:
...
-builtin flight mode controls. What happens is you setup the plane with
-correct trims in MANUAL by adjusting using the trip tabs in flight, but
-those trims don't get used when you change flight modes. You need to
-change your transmitter settings so that the stick inputs are the same
-in all flight modes.
+built in flight mode controls. You need to be sure  your transmitter trim settings 
+are the same, so that the stick inputs are the same in all flight modes and  match those
+you used for the RC calibration setup.
+
+Another, is that you are not using :ref:`SERVO_AUTO_TRIM<SERVO_AUTO_TRIM>` to automatically adjust the trim
```
Comment (kd0aij) [inline plane/source/docs/fixed-wing-faq.rst]: I haven't used servo_auto_trim before... What I have done is to use TX trims only for MANUAL mode, with trim set to zero for all other modes. Does auto_trim really eliminate the need for trimming in manual mode?  I assume you would at least have to fly in a stabilized mode first on any given day (with variations in temperature, etc.) for the trim to be set correctly.

Comment (Hwurzburg) [reply plane/source/docs/fixed-wing-faq.rst]: it works very well, as long as you want trim to be the attitude of level AHRS and at cruise adjusted throttle for stable altitude...STAB and MANUAL will fly just like FBWA/B/CRUISE..

I never touch trims...or worry about plane trim on first hand launch(I always AUTOTAKEOFF on maidens)....just cal at 1500us neutrals and use auto-trim...works great...

File: plane/source/docs/fixed-wing-faq.rst [hunk 2]
```diff
@@ -118,34 +118,39 @@ Please see :ref:`this page <automatic-landing>`
...
--  TECS_THR_DAMP is a damping factor for throttle control. The default
+-  :ref:`TECS_THR_DAMP<TECS_THR_DAMP>` is a damping factor for throttle control. The default
    is 0.5. A higher value will dampen throttle changes.
--  TECS_TIME_CONST is the overall time constant for both throttle and
+-  :ref:`TECS_TIME_CONST<TECS_TIME_CONST>` is the overall time constant for both throttle and
    pitch changes in TECS. It controls how rapidly TECS tries to correct
    for any error in speed or height. It is in seconds, and defaults to
-   5. A higher value makes the pitch and throttle corrections happen
+   5. A higher value makes the pitch and throttle corrections happen more
    more slowly.
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply plane/source/docs/fixed-wing-faq.rst]: fixed..thanks

**Comment by Hwurzburg** [comment]
@tridge I accidentally merged this with another tweak in a pre-coffee state this am....rather than removing it from the master, I will just make a separate fix for anything you dont like in this....if its okay, just close this....otherwise tell me what you want changed and I will submit another PR....thanks

**Pull request closed without merging**